site/public/js/server.js

```diff
@@ -754,6 +758,8 @@ function adminTeamForm(new_team, who_id, reg_section, rot_section, user_assignme
     var param = (new_team ? 3 : members.length+2);
     members_div.append('<span style="cursor: pointer;" onclick="addTeamMemberInput(this, '+param+');" aria-label="Add More Users"><i class="fas fa-plus-square"></i> \
         Add More Users</span>');
+    if (exists_multiple_invite_member) members_div.append('<div id="multiple_invites_warning" style="color:red;font-size:small;margin-top:3px;width:75%">\
```

Comment:
Use curly braces around javascript statements to improve readability and try to use existing CSS whenever possible, see `.red-message` class in `server.css` to make the warning message more consistent with the rest of Submitty